README.md

```diff
@@ -144,7 +144,8 @@ Key commands
 - <kbd>Ctrl</kbd> + <kbd>p</kbd> – Print to a PDF file using `gl2ps`. Other
   vector formats (SVG, EPS) are also possible, but keep in mind that the
   printing takes a while and the generated files are big.
-- <kbd>Q</kbd> – Cycle between representations of the quadrature (piece-wise constant refined/L2 element dof collocation) 
+- <kbd>Q</kbd> – Cycle between representations of the quadrature (piece-wise constant refined
+  / L2 element dof collocation / L2 element projection) 
```

Comment:
Wasn't `Q` supposed to close all GLVis windows? Or was that just an idea that never got implemented? @tzanio, do you remember?

Reply:
> Wasn't `Q` supposed to close all GLVis windows? Or was that just an idea that never got implemented? @tzanio, do you remember?

I don't think such a functionality was implemented.